modules/shared/facts.d/win_location_fact.ps1

```diff
@@ -0,0 +1,22 @@
+$DhcpDomain = ((Get-ItemProperty 'HKLM:SYSTEM\\CurrentControlSet\\Services\\Tcpip\\Parameters').'DhcpDomain')
```

Comment:
It is fine to leave this here for now but I would really like to have this written in ruby so we can use it with other non-windows OS like linux and macOS.  That way we can keep the use of determining dc/region data common to all things.

Reply:
Because of time I am going to continue to use powershell, but ultimately these will be converted to ruby.